### PR TITLE
:seedling: set work meta data to ManifestBundle event on source

### PR DESCRIPTION
**Pull request by skeeey**
## Related issue(s)

Fixes #

### CODE REVIEW

File: pkg/cloudevents/work/source/codec/manifestbundle.go [hunk 1]
```diff
@@ -15,6 +16,20 @@ import (
...
+	ExtensionWorkName = "name"
+
+	// ExtensionWorkLabels is an extension attribute for work labels.
+	ExtensionWorkLabels = "labels"
+
+	// ExtensionWorkAnnotations is an extension attribute for work annotations.
+	ExtensionWorkAnnotations = "annotations"
+
+	// ExtensionWorkFinalizers is an extension attribute for work finalizers.
+	ExtensionWorkFinalizers = "finalizers"
```
Comment (qiujian16) [inline pkg/cloudevents/work/source/codec/manifestbundle.go:30]: so what about we set whole metaObject as one value?

Comment (skeeey) [reply pkg/cloudevents/work/source/codec/manifestbundle.go:30]: let me think about this...

Comment (skeeey) [reply pkg/cloudevents/work/source/codec/manifestbundle.go]: using the whole metaObject, PTAL

**Review by qiujian16 — COMMENTED** [review]
/approve
/lgtm

**Pull request merged**